### ISSUE TITLE
Provide Accessioning Service Admin Team with Dashboard for Bulk Update/Action History

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/argo#991]
We are now providing users an extensive suite of bulk tools spread across the bulk update / bulk actions / metadata upload features. For reporting (and troubleshooting) purposes, it is highly desirable to have an admin dashboard that gives some of us permission-restricted access to the full list of bulk actions/updates/uploads.

This could be a view and/or report that provides a high-level view (including date, user - if known, action performed, number of objects, number of success, number of failures, and access to log (if available).

**Comment by jmartin-sul** [comment]
agreed that this would be very handy for reporting back to people with questions/complaints about jobs that seem slow/stalled/broken, and for initial troubleshooting of bulk action issues.
